src/services/lesson/docs/index.js

```diff
@@ -0,0 +1,36 @@
+module.exports = {
+	lessonService: {
+		description: "A service to do lessons",
```

Comment:
consistent: A service for lessons

Reply:
the naming convention of that service is anyway a bit odd.
We call it lesson, but in the end it's just a topic, but yeah a service doesn't do a lesson ^^